src/test/txrebroadcast_tests.cpp

```diff
@@ -57,6 +57,48 @@ BOOST_AUTO_TEST_CASE(recency)
     BOOST_CHECK_EQUAL(candidates.front().m_txid, tx_old.GetHash());
 }
 
+BOOST_AUTO_TEST_CASE(fee_rate)
```

Comment:
In 0ae1420767a320cfeafd2acee0b54bc6a6eb9b27 [test] Add unit test for the fee rate cache 

Would be nice if the test also checked what happens with transactions that are prioritized.